hedera-mirror-rest/controllers/contractController.js

```diff
@@ -1085,6 +1103,20 @@ class ContractController extends BaseController {
       fileData
     );
   };
+
+  getDefaultFailureContractResultByTransaction = (transaction) => {
+    return {
+      bloom: emptyBloomBuffer,
+      callResult: null,
+      createdContractIds: [],
+      functionParameters: [],
+      payerAccountId: transaction.payerAccountId,
+      errorMessage: TransactionResult.getName(transaction.result),
+      consensusTimestamp: transaction.consensusTimestamp,
+      contractId: transaction.entityId,
```

Comment:
Technically if there's no contract result there's no `transaction.entityId` since it's populated from the `record.contractFunctionResult.contractID` in the importer for EthereumTransaction. So it's a bit useless to return this in this scenario. You can see this reflected in the original ticket's JSON response:

https://testnet.mirrornode.hedera.com/api/v1/transactions/0.0.2859100-1655309548-529823609?nonce=0

Recommend you revert query returning entityId and set null here.

```suggestion
      contractId: null,
```

Reply:
Done